src/css/profile/mobile/themes/themes.less

```diff
@@ -40,7 +40,6 @@ body.ui-theme-dark {
     // eg. --primary-color: @primary-color;
     @import (multiple) "../common/oneui-common.less";
 
-    --text-color: @text-color;
```

Comment:
You cannot remove this line because you broke colors in other themes.
![image](https://user-images.githubusercontent.com/29534410/76501681-fdfc2200-6442-11ea-8df4-68f91c822b85.png)

Reply:
I miss to move it to the "onui-common.less".
I fixed it.